Products/CMFPlone/controlpanel/browser/usergroups_groupdetails.py

```diff
@@ -2,13 +2,23 @@
 from plone.base import PloneMessageFactory as _
 from plone.protect import CheckAuthenticator
 from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.controlpanel.browser.usergroups import (
-    UsersGroupsControlPanelView,
-)
+from Products.CMFPlone.controlpanel.browser.usergroups import UsersGroupsControlPanelView
+from Products.CMFPlone.PloneTool import PloneTool
 from Products.statusmessages.interfaces import IStatusMessage
 
-
 class GroupDetailsControlPanel(UsersGroupsControlPanelView):
+    def extract_invalid_emails(self, emails):
+        """Validate a list of email addresses."""
+        if isinstance(emails, str):
+            emails = [e.strip() for e in emails.split('\n') if e.strip()]
+        
```

Comment:
```suggestion
        
        plone_utils = getToolByName(self.context, "plone_utils")
```
You should use this rather than getting the class.